scripts/dev/download_release.sh

```diff
@@ -11,7 +11,7 @@ if [[ $# != 1 ]]; then
     exit 1
 fi
 
-cd "$tmpdir"
```

Comment:
Instead of adding all these `|| exit`'s, can we `set -e` at the top of the file to exit on any failing command?

Reply:
Sure, that makes sense. I'll do that.